apps/web/pages/team/[slug].tsx

```diff
@@ -138,9 +139,9 @@ function TeamPage({ team, isUnpublished, markdownStrippedBio, isValidOrgDomain }
                 className="mr-6"
                 size="sm"
                 truncateAfter={4}
-                items={ch.members
-                  .filter((mem) => mem.user.username !== null)
-                  .map(({ user: member }) => ({
+                items={team.members
+                  .filter((mem) => mem.subteams?.includes(ch.slug) && mem.accepted)
```

Comment:
We now rely on subteam slug matching iterated subteam and accepted property to show members in the profile, as the username will be set even for invited people